src/webui/www/private/views/preferences.html

```diff
@@ -2026,6 +2037,15 @@
 
                         // Advanced settings
                         // qBittorrent section
+                        switch (pref.resume_data_storage_type) {
+                            case "Legacy":
+                                index = 0;
+                                break;
+                            case "SQLite":
+                                index = 1;
+                                break;
+                        }
+                        $('resumeDataStorageType').getChildren('option')[index].setAttribute('selected', '');
```

Comment:
Couldn't it be written as before?

```js
$('resumeDataStorageType').setProperty('value', pref.resume_data_storage_type);
```

Reply:
Yes it seems. Other places use the current one though. Change it anyway?

Reply:
> Change it anyway?

I'm not familiar with the nuances of HTML/JS development, so I don't insist on anything.

Reply:
Ok changed, thanks.